src/cogent3/app/data_store_new.py

```diff
@@ -595,12 +598,12 @@ def convert_tinydb_to_sqlite(source: Path, dest: Optional[Path] = None) -> DataS
             f"Destination file {str(dest)} already exists. Delete or define new dest."
         )
     dstore = DataStoreSqlite(source=dest, mode=OVERWRITE)
-
+    writer = write_db(data_store=dstore)
     for id, data, is_completed in data_list:
-        if is_completed:
-            dstore.write(unique_id=id, data=data)
+        if id.endswith(".log"):
+            writer.data_store.write_log(unique_id=id, data=data)
         else:
-            dstore.write_not_completed(unique_id=id, data=data)
+            writer.main(data, identifier=id)
```

Comment:
@GavinHuttley 
Here, I am not sure writer works for not_completed members.

Reply:
@Nick-Foto It does, [see io_new](https://github.com/cogent3/cogent3/blob/a577676d3320ce4b6a4a10b41aa6be42025d9663/src/cogent3/app/io_new.py#L554) and [tests](https://github.com/cogent3/cogent3/blob/a577676d3320ce4b6a4a10b41aa6be42025d9663/tests/test_app/test_io_new.py#L455).

Reply:
@GavinHuttley 
Thank you.